mailpoet/lib/Segments/DynamicSegments/FilterDataMapper.php

```diff
@@ -125,6 +126,16 @@ private function createSubscriber(array $data): DynamicSegmentFilterData {
       if (!empty($data['operator'])) $filterData['operator'] = $data['operator'];
       return new DynamicSegmentFilterData(DynamicSegmentFilterData::TYPE_USER_ROLE, $data['action'], $filterData);
     }
+    if ($data['action'] === SubscriberTag::TYPE) {
```

Comment:
Seems like this method (`FilterDataMapper::map`) needs tests for all branches (and for this new branch) especially since it's rather big and convoluted, what do you think?

Reply:
Good point. I will extend our tests.